helm/ngrok-operator/values.yaml

```diff
@@ -195,27 +188,74 @@ serviceAccount:
   name: ""
   annotations: {}
 
-
-## Logging configuration
+##
+## @section Logging configuration
+##
 ## @param log.level The level to log at. One of 'debug', 'info', or 'error'.
 ## @param log.stacktraceLevel The level to report stacktrace logs one of 'info' or 'error'.
 ## @param log.format The log format to use. One of console, json.
+##
 log:
   format: json
   level: info
   stacktraceLevel: error
 
-## @param lifecycle an object containing lifecycle configuration
-## ref: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
 ##
-lifecycle: {}
+## @section Credentials configuration
+##
+## @param credentials.secret.name The name of the secret the credentials are in. If not provided, one will be generated using the helm release name.
+## @param credentials.apiKey Your ngrok API key. If provided, it will be will be written to the secret and the authtoken must be provided as well.
+## @param credentials.authtoken Your ngrok authtoken. If provided, it will be will be written to the secret and the apiKey must be provided as well.
+##
+credentials:
+  secret:
+    name: ""
+  apiKey: ""
+  authtoken: ""
 
+##
+## @section Kubernetes Ingress feature configuration
+##
+## @extra ingressClass.name DEPRECATED: Use ingress.ingressClass.name instead
+## @extra ingressClass.create DEPRECATED: Use ingress.ingressClass.create instead
+## @extra ingressClass.default DEPRECATED: Use ingress.ingressClass.default instead
+## @extra watchNamespace DEPRECATED: Use ingress.watchNamespace instead
+## @extra controllerName DEPRECATED: Use ingress.controllerName instead
+## @param ingress.enabled When true, enable the Ingress controller features
+## @param ingress.ingressClass.name The name of the ingress class to use.
+## @param ingress.ingressClass.create Whether to create the ingress class.
+## @param ingress.ingressClass.default Whether to set the ingress class as default.
+## @param ingress.watchNamespace The namespace to watch for ingress resources (default all)
+## @param ingress.controllerName The name of the controller to look for matching ingress classes
+##
+ingress:
+  enabled: true # enabled by default
+  controllerName: "k8s.ngrok.com/ingress-controller"
+  watchNamespace: "" # default all
+  ingressClass:
```

Comment:
My preferenc is still ingressClass since that's the type of resource being created: https://github.com/ngrok/ngrok-operator/blob/main/helm/ngrok-operator/templates/ingress-class.yaml#L3